pkg/kube/inject/testdata/inject/explicit-security-context.yaml.injected

```diff
@@ -177,7 +177,7 @@ spec:
           runAsNonRoot: false
           runAsUser: 0
       securityContext:
-        fsGroup: 1337
+        fsGroup: 1234
```

Comment:
curious why the change to 1234 here but not seeing a corresponding verification change?

Reply:
```
pkg/kube/inject/testdata/inject/explicit-security-context.yaml
19:        fsGroup: 1234
```

I think this is WAI. Before we had to override users fsGroup, now we let it remain